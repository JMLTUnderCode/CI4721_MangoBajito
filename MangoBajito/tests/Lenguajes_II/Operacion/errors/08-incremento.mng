# Test 08: Operacion de errores en incremento.
se_prende(){
	culito decl0 : negro;
	decl0++;

	culito decl1 : higuerote;
	decl1++;

	culito decl2 : tas_claro;
	decl2++;
};