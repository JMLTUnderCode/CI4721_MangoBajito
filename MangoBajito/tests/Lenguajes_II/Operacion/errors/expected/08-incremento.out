| AcomÃ³date que te lo estan analizando d:-)
Error: NON_VALUE
  - Sendo peo en la linea 4, columna 7: Esta vaina "decl0" tiene el amor que te diÃ³ ella, o sea vacÃ­o.
  - Sendo peo en la linea 7, columna 7: Esta vaina "decl1" tiene el amor que te diÃ³ ella, o sea vacÃ­o.
  - Sendo peo en la linea 10, columna 7: Esta vaina "decl2" tiene el amor que te diÃ³ ella, o sea vacÃ­o.
Error: TYPE_ERROR
  - Sendo peo en la linea 13, columna 7: Tas en droga?. Tienes "decl3" de tipo 'negro' y debe ser de tipo 'mango' | 'manguita' | 'manguangua', locota.
  - Sendo peo en la linea 16, columna 7: Tas en droga?. Tienes "decl4" de tipo 'higuerote' y debe ser de tipo 'mango' | 'manguita' | 'manguangua', locota.
  - Sendo peo en la linea 19, columna 7: Tas en droga?. Tienes "decl5" de tipo 'tas_claro' y debe ser de tipo 'mango' | 'manguita' | 'manguangua', locota.
[1;31m[5m
               =======================================================
                       ---->        Error Program        <----        
               =======================================================
[0m
| Hay un poquito de cÃ³digo en tu marihuana d:-]
